### DISCUSSION TITLE
The ability to terminate a circuit to a location within a site

### DISCUSSION THREAD

**Discussion by mkarel** [netbox-community/netbox#12754] [Ideas]
Today in version 3.5.2 it looks like you can terminate a circuit to a site but not to a location. it would be great if we could add the additional granularity of the location of the termination within the site.  Example, we have a site say our Hospital campus with 2 locations a data center and an MDF. we would like to be able to indicate what location within a site the termination is.

**Comment by GoldyXXD** [comment]
Looks like they've already been discussing it.
https://github.com/netbox-community/netbox/issues/9604
